scan/src/components/TxsTable.re

```diff
@@ -50,42 +57,66 @@ let make = (~txs: list(TxHook.Tx.t)) => {
     </THead>
     {txs
      ->Belt.List.map(({blockHeight, hash, fee, messages, success}) => {
+         let numMsgs = messages->Belt_List.size;
          <TBody key={hash |> Hash.toHex}>
            <Row minHeight={`px(30)}>
              <HSpacing size={`px(20)} />
-             <Col size=1.67> <TxLink txHash=hash width=140 /> </Col>
-             <Col size=0.88> <TypeID.Block id={ID.Block.ID(blockHeight)} /> </Col>
+             <Col size=1.67>
+               <div className={Styles.heightByMsgsNum(numMsgs, 0)}>
+                 <TxLink txHash=hash width=140 />
+               </div>
+             </Col>
+             <Col size=0.88>
+               <div className={Styles.heightByMsgsNum(numMsgs, -4)}>
```

Comment:
Why it must be -4 and if more msg this margin would be scale follow a number of messages?

Reply:
The good way to do this would need more time. So right now will fix it by using magic number. And we might come back to fix it if needed.